guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -9,7 +9,16 @@ There are no highlights with Katello {KatelloVersion}.
 [id="katello-upgrade-warnings"]
 == Upgrade Warnings
 
-There are no upgrade warnings with Katello {KatelloVersion}.
+#There are no upgrade warnings with Katello {KatelloVersion}.
```

Comment:
```suggestion
// There are no upgrade warnings with Katello {KatelloVersion}.
```

comments in asciidoc: https://docs.asciidoctor.org/asciidoc/latest/comments/#comment-lines